.github/workflows/release.yml

```diff
@@ -53,7 +52,17 @@ jobs:
         if: steps.decide_whether_to_release.outputs.conclusion
         run: ./gradlew jpackage
 
-      - name: Release
+      - name: Release JAR
+        if: steps.decide_whether_to_release.outputs.conclusion == 'true' && matrix.os == 'ubuntu-latest'
+        uses: softprops/action-gh-release@v1
+        with:
+          name: ${{ steps.decide_whether_to_release.outputs.name }}
+          tag_name: ${{ steps.decide_whether_to_release.outputs.tag_name }}
+          prerelease: ${{ steps.decide_whether_to_release.outputs.prerelease }}
+          generate_release_notes: ${{ steps.decide_whether_to_release.outputs.gen_rel_notes }}
+          files: build/libs/*.jar
```

Comment:
@vedantmgoyal2009 I missed this but this should be `build/libs/*-all.jar`. The normal Jar doesn't contain dependencies.